sunsynk/definitions.py

```diff
@@ -108,9 +108,9 @@
     Sensor(108, "Day PV Energy", KWH, 0.1),
     Sensor(61, "Day Reactive Energy", "kVarh", -0.1),
     # Sensor((201, 202), "History Load Power", KWH, 0.1),
-    Sensor(67, "Month Grid Energy", KWH),
-    Sensor(66, "Month Load Energy", KWH),
-    Sensor(65, "Month PV Energy", KWH),
+    Sensor(67, "Month Grid Energy", KWH, 0.1),
```

Comment:
Can you remove these changes from this PR? (It should already be part of your first PR)